pages/journal/index.js

```diff
@@ -0,0 +1,88 @@
+import Link from "next/link";
+import styled from "styled-components";
+
+function Journal({ journalData, journals }) {
+  return (
+    <>
+      <JournalWrapper>
+        <h1>Journal</h1>
+        <EntriesWrapper>
+          {journals.length > 0 ? (
+            <>
+              {journals.map((journal) => (
+                <JournalLink key={journal.id} href={`/journal/${journal.id}`}>
+                  {journal.name} in {journal.destination}
+                </JournalLink>
+              ))}
+            </>
+          ) : (
+            "No Journals yet. Create one!"
+          )}
+          <NewJournalButton href="/journal/form">New Journal</NewJournalButton>
+        </EntriesWrapper>
+        <DashboardLink href="/">Dashboard</DashboardLink>
+      </JournalWrapper>
+    </>
```

Comment:
Just deleted the fragment:
```suggestion
      <JournalWrapper>
        <h1>Journal</h1>
        <EntriesWrapper>
          {journals.length > 0 ? (
            <>
              {journals.map((journal) => (
                <JournalLink key={journal.id} href={`/journal/${journal.id}`}>
                  {journal.name} in {journal.destination}
                </JournalLink>
              ))}
            </>
          ) : (
            "No Journals yet. Create one!"
          )}
          <NewJournalButton href="/journal/form">New Journal</NewJournalButton>
        </EntriesWrapper>
        <DashboardLink href="/">Dashboard</DashboardLink>
      </JournalWrapper>
```